pkg/runtime/serializer/recognizer/recognizer.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package recognizer
```

Comment:
Will do - this is just a rough cut refactor for interface concept agreement.


On Dec 2, 2015, at 4:59 PM, Daniel Smith <notifications@github.com> wrote:

In pkg/runtime/serializer/recognizer/recognizer.go
<https://github.com/kubernetes/kubernetes/pull/17922#discussion_r46481894>:

> +Copyright 2014 The Kubernetes Authors All rights reserved.
> +
> +Licensed under the Apache License, Version 2.0 (the "License");
> +you may not use this file except in compliance with the License.
> +You may obtain a copy of the License at
> +
> +    http://www.apache.org/licenses/LICENSE-2.0
> +
> +Unless required by applicable law or agreed to in writing, software
> +distributed under the License is distributed on an "AS IS" BASIS,
> +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> +See the License for the specific language governing permissions and
> +limitations under the License.
> +*/
> +
> +package recognizer

I see, so it wraps decoders? That's fine, a comment would be helpful

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/17922/files#r46481894>.